### DISCUSSION TITLE
Add link to GitHub Change-log to Mobile app

### DISCUSSION THREAD

**Discussion by ChrisPearce** [community/community#16428] [Mobile]
It would be great to see the GitHub change-log surfaced in the mobile app.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.64.0 (66902451)
Device: iPhone 13 Pro (iOS 15.5)
</details>

**Comment by brianlovin** [comment]
+1, I would love for us to have this as well — we tried something a long time ago, but couldn't find a simple enough way to maintain it. I'll add your vote to our tracking issue for this 😊
